front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -225,6 +225,7 @@ const AdminProjectsProjectGeneral = () => {
   const handleProjectCardImageOnAdd = (projectImages: UploadFile[]) => {
     setSubmitState('enabled');
     setProjectCardImage(projectImages[0]);
+    setCroppedProjectCardBase64(projectImages[0].base64);
   };
```

Comment:
The problem here is that when we finally save a new image we use the value from croppedProjectCardBase64. However if we only add a new image and don't move it around, this cropped value base64 is never set.